src/composables/useSpacefinderSupabase.ts

```diff
@@ -32,5 +34,56 @@ export default function useSpacefinderSupabase() {
     buildingsRealtimeChannel.subscribe();
   }
 
-  return { getBuildingsOccupancyCurrent, subscribeToBuildingsOccupancy };
+  async function getSpacesOccupancyCurrent() {
+    const { data, error, status } = await client
```

Comment:
Should we also handle `error` and `status`?

Reply:
I'll create a separate ticket for this, removed the unused bindings for now